app/apps/visits/serializers.py

```diff
@@ -6,3 +6,13 @@ class VisitSerializer(serializers.ModelSerializer):
     class Meta:
         model = Visit
         fields = "__all__"
+
+
+class VisitRelatedSerializer(serializers.ModelSerializer):
+    """
+    Serializer for Visit m2m relation but with start_time added
```

Comment:
I think it should be many-to-one instead of m2m in the comment here. (Visit has a ForeignKey reference to ItineraryItem which is many-to-one)

Reply:
Scherp!